### PR TITLE
Implement smooth scrolling after updating an item

### PR DESCRIPTION
**Pull request by dariasamo**
## Description of changes

scrollTo() method replaced by smoothScrollTop() function

## Related issues (if any)

https://trello.com/c/KIsSWQEG/12-implement-smooth-scrolling-after-updating-an-item

## Testing
No related test created. Passed linter.
- [ ] Please confirm `npm run test-all` ran successfully.

### CODE REVIEW

File: admin/client/App/screens/Item/components/EditForm.js [hunk 1]
```diff
@@ -131,8 +131,14 @@ var EditForm = React.createClass({
...
 
 		list.updateItem(data.id, formData, (err, data) => {
-			// TODO: implement smooth scolling
-			scrollTo(0, 0); // Scroll to top
+			function smoothScrollTop () {
+				if (document.body.scrollTop || document.documentElement.scrollTop) {
+					window.scrollBy(0, -50);
+					var timeOut = setTimeout(smoothScrollTop, 20);
+				}
+				else clearTimeout(timeOut);
```
Comment (mxstbr) [inline admin/client/App/screens/Item/components/EditForm.js]: nit: please put all blocks in `{}`, i.e.

```JS
} else {
  clearTimeout(timeOut);
}
```

File: admin/client/App/screens/Item/components/EditForm.js [hunk 2]
```diff
@@ -131,8 +131,14 @@ var EditForm = React.createClass({
 		});
 
 		list.updateItem(data.id, formData, (err, data) => {
-			// TODO: implement smooth scolling
-			scrollTo(0, 0); // Scroll to top
+			function smoothScrollTop () {
```
Comment (mxstbr) [inline admin/client/App/screens/Item/components/EditForm.js]: nit: please move this function to the top level in this file

**Comment by mxstbr** [comment]
Two tiny nitpicks, but this looks amazing, good job!

**Comment by mxstbr** [comment]
Awesome, thanks!

**Comment by dariasamo** [comment]
Thanks for feedback!

**Pull request merged**